modules/cert-manager-uninstall-console.adoc

```diff
@@ -13,10 +13,53 @@ You can uninstall the {cert-manager-operator} by using the web console.
 * You have access to the cluster with `cluster-admin` privileges.
 * You have access to the {product-title} web console.
 * The {cert-manager-operator} is installed.
+* You have a backup of cluster issuers and certificates.
 // TODO: Any other prereqs, like removing anything that is using it?
 
 .Procedure
 
+. Get a list of issuers by running the following command:
++
+[source,terminal]
+----
+$ oc get issuers.cert-manager.io -A
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE      NAME                               READY   AGE
+cert-manager   letsencrypt-issuer                 True    19h
+----
+
+. Get a list of cluster issuers by running the following command:
++
+[source,terminal]
+----
+$ oc get clusterissuers.cert-manager.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                        READY   AGE
+letsencrypt-clusterissuer   True    19h
+----
+
+. You can get a list of certificates by running the following command:
++
+[source,terminal]
+----
+$ oc get certificates.cert-manager.io -A
```

Comment:
Since we're adding a get over ClusterIssuer, Issuer;
I think we should also do get over Certificate(s) of all namespaces as that is probably the most important resource managed by cert-manager.
```
$ oc get certificates -A
```